README.md

```diff
@@ -237,6 +237,55 @@ spec:
 
 Create a PVC using the storage class created for the ZFS driver.
 
+1. If storageclass has fstype as ext4/3/2, xfs, btrfs then we have to create the PVC with storage size which is multiple of volblocksize mentioned in the storageclass. The same is also true if fstype is not mentioned in the storageclass in which case dafault will be ext4.
+
+```
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: openebs-zfspv
+parameters:
+  volblocksize: "4k"
+  fstype: "ext4"
+  poolname: "zfspv-pool"
+provisioner: zfs.csi.openebs.io
+```
+
+Here, PVC should have the storage multiple of volblocksize which is 4k (4 * 1024). If is not the multiple of volblocksize then ZFS will not create the volume and pod will not able able to come into RUNNING state as mount will keep on failing. For example, if we have created the below PVC with the above storageclass
+
+```
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: csi-zfspv
+spec:
+  storageClassName: openebs-zfspv
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 4G
+```
+
+Here note that storage is 4G. The volume size requested in PVC 4G (4 * 1000 * 1000 * 1024) is not multiple of 4k (4 * 1024, for zfs k is 1024). So, volume creation will fail. We can fix it by changing the storage size to 4Gi which is 4 * 1024 * 1024 * 1024, which is multiple of 4k (4 * 1024).
+
+
+If volblocksize is not mentioned in the storageclass then ZFS takes 8k as the default volblocksize and PVC size should be multiple of 8k (8 * 1024).
+
+```
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: openebs-zfspv
+parameters:
+  poolname: "zfspv-pool"
+provisioner: zfs.csi.openebs.io
+```
+
+Here PVC size should be multiple of 8k (8 * 1024).
+
+2. If storageclass has fstype as "zfs" then we don't need to worry about the size. We can use any size we want as in this case a dataset will be created not zvol.
```

Comment:
for zfs we can add, that recordsize field should be in power of 2.

Reply:
as this section for PVC, added that detail at the other place.